package.json

```diff
@@ -14,7 +14,8 @@
     "e2e": "ng e2e",
     "extract": "ngx-translate-extract --input ./src --output ./src/assets/i18n/en.json ./src/assets/i18n/es.json --clean --sort --format namespaced-json --marker _",
     "firebase:deploy": "firebase deploy",
-    "firebase-db:override-keys": "echo $FIREBASE_DB_KEYS > ./src/environments/firebase-config.ts"
+    "env-keys-config:override-keys": "echo $KEYS_CONFIG > ./src/environments/keys-config.ts",
+    "sendgrid:override-keys": "echo $SEND_GRID_KEY > ./functions/.runtimeconfig.json"
```

Comment:
Not sure if this is necessary as the environment variables for the cloud functions are created by the command line, the .runtimeconfig.json is only used for local testing. I already setup this for the gp3techsite project.

https://firebase.google.com/docs/functions/config-env

Reply:
@dmeneses I removed the line, please review again